mirascope/base/extractors.py

```diff
@@ -224,7 +227,10 @@ async def _extract_attempt_async(
                 raise
 
         if isinstance(retries, int):
-            retries = AsyncRetrying(stop=stop_after_attempt(retries))
+            if retries > 1:
```

Comment:
should this be `> 0` everywhere?

If a user passes in `Retrying` with retries set to `1` it will now behave differently from passing in `retries=1` right? I think we want to match that behavior but default to `0` where `0` means no tenacity?

Reply:
thats fine we can do that too